doc/conf.py

```diff
@@ -288,52 +287,78 @@
 htmlhelp_basename = 'PennyLaneQiskitdoc'
 
 # # -- Xanadu theme ---------------------------------------------------------
-html_theme = 'xanadu_theme'
+html_theme = 'xanadu'
 html_theme_path = ['.']
 
 # Register the theme as an extension to generate a sitemap.xml
 # extensions.append("guzzle_sphinx_theme")
 
 # xanadu theme options (see theme.conf for more information)
 html_theme_options = {
-
-    # Set the path to a special layout to include for the homepage
-    # "index_template": "special_index.html",
-
-    # Set the name of the project to appear in the left sidebar.
-    "project_nav_name": "PennyLane-Qiskit",
-
-    # Set your Disqus short name to enable comments
-    # "disqus_comments_shortname": "pennylane-1",
-
-    # Set you GA account ID to enable tracking
-    "google_analytics_account": "UA-130507810-2",
-
-    # Path to a touch icon
-    "touch_icon": "logo_new.png",
-
-    # Specify a base_url used to generate sitemap.xml links. If not
-    # specified, then no sitemap will be built.
-    # "base_url": ""
-
-    # Allow a separate homepage from the master_doc
-    # "homepage": "index",
-
-    # Allow the project link to be overriden to a custom URL.
-    # "projectlink": "http://myproject.url",
-
-    "large_toc": True,
-    # colors
-    "navigation_button": "#19b37b",
-    "navigation_button_hover": "#0e714d",
-    "toc_caption": "#19b37b",
-    "toc_hover": "#19b37b",
-    "table_header_bg": "#edf7f4",
-    "table_header_border": "#19b37b",
-    "download_button": "#19b37b",
-    # gallery options
-    # "github_repo": "PennyLaneAI/PennyLane",
-    # "gallery_dirs": "tutorials",
+    "navbar_wordmark_path": "_static/pennylane.png",
```

Comment:
We should copy over `pl_wordmark.png` from [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/master/doc/_static/pl_wordmark.png) since that image is scaled to fit the navbar better.

Reply:
Ok I will!